### ISSUE TITLE
Can not get PhysXArticulations20201 branch working

### ISSUE THREAD

**Issue by Sohojoe** [Unity-Technologies/ml-agents#3109]
I am experimenting with the new PhysX Articulations in Unity 2020.1 - when I pull the PhysXArticulations20201 branch and run, I get the following error:

```
Assembly 'Assets/ML-Agents/Plugins/ProtoBuffer/Grpc.Core.dll' will not be loaded due to errors:
Grpc.Core references strong named System.Interactive.Async, versions has to match. Assembly references: 3.0.1000.0 Found in project: 3.0.3000.0.
```

Unity v 2020.1.0a16

FYI @VilmantasBal

**Comment by Sohojoe** [comment]
As a temp HACK - replacing System.Interactive.Async with v3.0.1 seams to work:

[System.Interactive.Async.dll.zip](https://github.com/Unity-Technologies/ml-agents/files/3991729/System.Interactive.Async.dll.zip)

**Comment by nukadelic** [comment]
Same in 2020.1.0a17, tried adding the following the the `csproj` file with now luck:
```
  <dependentAssembly>
    <assemblyIdentity name="System.Reactive.Interfaces" publicKeyToken="94bc3704cddfc263" culture="neutral" />
    <bindingRedirect oldVersion="0.0.0.0-3.0.3000.0" newVersion="3.0.1000.0" />
  </dependentAssembly>
```
however @Sohojoe dll replacement file does the trick

**Comment by surfnerd** [comment]
We've updated the System.Interactive.Async assembly to resolve this issue.  The fix was merged to master in #3512 and will be in the next release.
